src/_igraph/graphobject.c

```diff
@@ -12902,6 +12981,39 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  multiplying by the number of vertices minus one.\n"
    "@return: the calculated closenesses in a list\n"},
 
+  /* interface to igraph_harmonic_centrality */
+  {"harmonic_centrality", (PyCFunction) igraphmodule_Graph_harmonic_centrality,
+   METH_VARARGS | METH_KEYWORDS,
+   "harmonic_centrality(vertices=None, mode=ALL, cutoff=None, weights=None,\n"
+   "          normalized=True)\n\n"
+   "Calculates the harmonic centralities of given vertices in a graph.\n\n"
+   "The harmonic centerality of a vertex measures how easily other\n"
+   "vertices can be reached from it (or the other way: how easily it\n"
+   "can be reached from the other vertices). It is defined as the\n"
+   "number of vertices minus one times the sum of one divided by \n"
+   "the length of each geodesic from/to the given vertex.\n\n"
```

Comment:
This is a bit hard to read for me ... I would say that the harmonic centrality of a vertex is the mean _inverse distance_ to/from all other vertices.

Reply:
yep, done